workbench-core/example/infrastructure/integration-tests/support/complex/datasetHelper.ts

```diff
@@ -50,14 +53,26 @@ export class DatasetHelper {
   }
 
   public async deleteDdbRecords(dataSetId: string): Promise<void> {
```

Comment:
is this used for clean up?

Reply:
yes, in the `cleanup` task of the `Dataset` resource.